submissions/peterIyiola/database/init.sql

```diff
@@ -0,0 +1,132 @@
+values('samsung 55 inches qled','smart,android enabled Tv','https://github.com/peteriyiola/Dufuna-CodeCamp20/blob/master/submissions/peterIyiola/htmlForms/images/samsung2.jpg','500000.00', '2', '4',' 180', 'in stock'),
+('LG 55 inches qled','smart,android enabled Tv','https://github.com/peteriyiola/Dufuna-CodeCamp20/blob/master/submissions/peterIyiola/htmlForms/images/samsung2.jpg','400000.00',' 5', '4', '120', 'in stock'),
+('Hisense 55 inches qled','smart,android enabled Tv','https://github.com/peteriyiola/Dufuna-CodeCamp20/blob/master/submissions/peterIyiola/htmlForms/images/samsung2.jpg','300000.00', '6', '4', '130', 'in stock'),
+('Panasonic 55 inches qled','smart,android enabled Tv','https://github.com/peteriyiola/Dufuna-CodeCamp20/blob/master/submissions/peterIyiola/htmlForms/images/samsung2.jpg','200000.00', '4', '4', '150', 'in stock'),
+('Thermocool 55 inches qled','smart,android enabled Tv','https://github.com/peteriyiola/Dufuna-CodeCamp20/blob/master/submissions/peterIyiola/htmlForms/images/samsung2.jpg','300000.00',' 3', '4', '160', 'in stock');
+
+/--- created customers table --/
+create table customers (
+	id int not null auto_increment,
+	first_name varchar(40) NOT NULL,
+	last_name varchar(20) NOT NULL,
+	email_address varchar(50) NOT NULL,
+	password varchar(30) NOT NULL,
+	PRIMARY KEY (id)
+);
+
+/--- insert values into customers  --/
+insert into customers (first_name, last_name ,email_address, password)
+values('Patrick', 'Isreal','patricki@gmail.com','p745377'),
+('tommy' ,'hagen','tm@gmail.com','TM52637'),
+('Jamie' ,'vardy','jv@gmail.com','jvte7e878832'),
+('richard' ,'Hanger','rm@gmail.com','TM52yry73'),
+('Samuel' ,'etuhu','se@gmail.com','TM5=7445');
+
+/--- create table for customer details --/
+create table customer_addresses (
+	id int not null auto_increment,
+	customer_id int null,
+	street_name varchar(100) not null,
+	city varchar(20) not null,
+	state varchar(20) not null,
+	zip_code int,
+	country varchar(20) not null,
+	phone_number varchar(15) not null,
+    primary key (id),
+	foreign key (customer_id) references customers (id) on delete cascade
+);
+
+insert into customer_addresses (customer_id, street_name,city,state,zip_code,country,phone_number)
+values('1','abejoye','VI','lagos','100449','nigeria','08074949934'),
+('2','akinlabake','ishaga','lagos','100949','nigeria','08074945930'),
+('3','akerele','ikotun','lagos','100449','nigeria','08073949935'),
+('4','adefemi','gbagada','lagos','100679','nigeria','08079949930'),
+('5','adebayo','ipaja','lagos','100959','nigeria','0807594930');
+
+/--- create table for orders --/
+CREATE TABLE orders (
+	id int not null auto_increment,
+	customer_orders int not null,
+	order_amount decimal(9,2) not null,
+	order_created_at datetime,
+	primary key (id),
+	foreign key (customer_orders) references customers (id) on delete cascade
+);
+
+insert into orders (customer_orders, order_amount, order_created_at)
+values('1', '350000.00', now()),
```

Comment:
This shouldn't be around quotes `'350000.00'`. It should be this `350000.00`. Same goes for others